src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,25 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
 public class Main {
+
+    private static Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
 
+        MovieService movieService =
+                (MovieService) injector.getInstance(MovieService.class);
+
+        Movie movie = new Movie();
+        movie.setTitle("Terminator 2");
+        movie.setDescription("Awesome movie");
+        movieService.add(movie);
+
+        Movie movieFromDB = movieService.get(1L);
```

Comment:
```suggestion
        Movie movieFromDB = movieService.get(movie.getId());
```
the id will be set under the hood to the movie object